doc/ref_cert/RC2/chapters/chapter01.md

```diff
@@ -51,19 +50,24 @@ Perform Kubernetes infrastructure and CNF Verification and Validations using CNT
 <a name="1.1.1"></a>
 ### 1.1.1 Terminology
 
-Terminology in this document will follow [CNTT Terminology](../../../tech/glossary.md).
+Terminology in this document will follow [CNTT Terminology](../../../tech/glossary.md). Relevant testing terminology includes:
+
+**Functional Testing**: Most but not all the existing tests currently being considered for RC-2 are functional tests. Reuse of existing test suites is encouraged however OVP 2.0 is not aiming to replace programs such as Certified Kubernetes which is a "CNCF software conformance program that ensures a vendor’s version of Kubernetes supports the required APIs, as do open source community version". Testing to confirm specific versions of APIs or software components is out of scope of conformance sine this is an unambiguous part of a product specification. The focus of functional testing in conformance is the actual behavior of the system wrt a capability specified in the RM or RA-2.
+
+**Performance Testing**: The main objective of CNTT performance testing is to understand if a potential BOM is functioning correctly to get expected performance. It requires a set of performance testing tools (open source) that help with the dimensioning of a solution by an operator. This testing is a comparison between the SUT and a CNTT reference implementation with (pass/fail) result. Performance testing for comparisons between commercial implementations is not a goal of CNTT. Performance testing relies on existing benchmarks.
+
+**Benchmarking**: Benchmarking assessments do not define acceptance criteria or numerical performance requirement. Benchmark testing and Conformance testing intersect when a specific requirement in the software specification is very important, such as a frequently-executed function. Correct execution with the expected result constitutes conformance. The completion time for a single conforming execution, or the number of conforming executions per second are potential Benchmarks. Benchmarks assess a key aspect of the computing environment in its role as the infrastructure for cloud-native network functions. The benchmarks (and related metrics) have been agreed by the Industry and documented in publications of an accredited standards body. As a result, benchmarks are a sub-set of all possible performance metrics. Examples benchmarks include data rate, latency, and loss ratio of various components of the environment, expressed in quantitative units to allow direct comparison between different systems treated as a black box (vendor-independence). Because the demands on a particular system may vary from deployment to deployment, Benchmarking assessments do not define acceptance criteria or numerical performance requirements.
+
```

Comment:
can we keep this as a separate PR under #1495 ? if this is removed, I'm fine with merging the rest